docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -75,13 +75,13 @@ Note that these inconsistencies are limited to the tables/rows being written to
 
 This mode is an extension of the previous one.  In order to restore consistency, we additionally disallow writes on the target universe and cause reads to read as of a time far enough in the past that all the relevant data from the source universe has already been replicated.
```

Comment:
as of a time far enough in the past (typically 250ms)

Reply:
Its good to have an estimate.

Reply:
:+1: